docs/guides/configure-error-suppression.mdx

```diff
@@ -126,11 +130,11 @@ psi1_H1 = job.result()
 
 
 <span id="transpilation-table"></span>
-## Advanced runtime compilation options
+## Advanced error suppression options
 
 <Tabs>
   <TabItem value="PrimV2" label="V2 primitives">
-    In the V2 primitives, you can explicitly enable and disable individual error mitigation/suppression methods, such as dynamical decoupling.
+    In the V2 primitives, you can explicitly enable and disable individual error mitigation and suppression methods, such as dynamical decoupling. See [Advanced Qiskit Runtime options](runtime-options-overview) for information about available options.
```

Comment:
I like this points to the overview table instead of the API reference. Maybe the error mitigation page can also do the same? (under "Custom error settings (V2 primitives)")

```suggestion
    In the V2 primitives, you can explicitly enable and disable individual error mitigation and suppression methods, such as dynamical decoupling. See [Available options](runtime-options-overview#available-options) for information about available options.
```